level-4/올바른-괄호의-갯수&12929&.js

```diff
@@ -0,0 +1,10 @@
+//https://github.com/codeisneverodd/programmers-coding-test
+//완벽한 정답이 아닙니다.
+//정답 1 - cg10036
+function fact(n) {
+    return n ? BigInt(n) * fact(n - 1) : 1n;
+}
+
+function solution(n) {
+    return fact(n * 2) / (fact(n) * fact(n + 1));
+}
```

Comment:
너무 깔끔한 풀이네요 👍👍
`fact` 라는 함수가 `solution` 내에서만 쓰인다면 내부로 위치를 옮겨보는 것은 어떨까요?
 
```suggestion
function solution(n) {
    const fact = n => n ? BigInt(n) * fact(n - 1) : 1n;
    return fact(n * 2) / (fact(n) * fact(n + 1));
}
```